marimba-web/Models/Term.cs

```diff
@@ -9,10 +9,39 @@ public class Term
     {
         public string name { get; set; } // i.e. W2020
         public IList<Member> members { get; set; }
-        public IList<Member> limboMembers { get; set; }
+        public IList<Member> limboMembers { get; set; } // Subset of members
         public IList<Rehearsal> rehearsals { get; set; }
         public DateTime startDate { get; set; }
         public DateTime endDate { get; set; }
         public decimal feeAmount { get; set; }
+
+        public Dictionary<Guid, Member> ConstructMembersDict()
+        {
+            Dictionary<Guid, Member> memberDict = new Dictionary<Guid, Member>();
+
+            foreach (var m in members)
+            {
+                memberDict.Add(m.id, m);
+            }
+
+            return memberDict;
+        }
+
+        public HashSet<Guid> ConstructLimboMemberIds()
```

Comment:
This suggestion depends on the outcome of the discussion about how limbo members are stored in Term.cs.

Reply:
It is a bit awkward, I used a set for limbo members because I only needed their IDs. Alternatively I could use another dictionary again and just get the keys as a set.